angular/projects/admin-nrpti/src/app/records/administrative-penalties/administrative-penalty-add-edit/administrative-penalty-add-edit.component.ts

```diff
@@ -542,6 +543,11 @@ export class AdministrativePenaltyAddEditComponent implements OnInit, OnDestroy
     // tslint:disable-next-line:max-line-length
     this.myForm.get('legislations').dirty &&
       (administrativePenalty['legislation'] = this.parseLegislationsFormGroups());
+    // swapping legislation with actCode
+    const actTitle = administrativePenalty['legislation'][0]['act'];
```

Comment:
Do we need to check for existence of `administrativePenalty['legislation'` before indexing?

Reply:
see below